api/app/db/models/hfi_calc.py

```diff
@@ -58,6 +58,7 @@ class PlanningWeatherStation(Base):
     __table_args__ = (
         UniqueConstraint('station_code', 'planning_area_id',
                          name='unique_station_code_for_planning_area'),
+        UniqueConstraint('order_of_appearance_in_planning_area_list', 'planning_area_id', name='unique_order_for_planning_area'),
```

Comment:
Shouldn't this be a `UniqueConstraint` on `order_of_appearance_in_planning_area_list`? Maybe also it is a constraint on `[station_code, planning_area_id, order_of_appearance_in_planning_area_list]` that way a station that could potentially be in multiple planning areas can have different orders.